pkgs/development/interpreters/python/cpython/3.4/default.nix

```diff
@@ -39,6 +39,8 @@ let
     ++ optionals x11Support [ tcl tk libX11 xproto ]
     ++ optionals stdenv.isDarwin [ CF configd ];
 
+  hasDistutilsCxxPatch = !(stdenv.cc.isGNU or false);
```

Comment:
How about `needsDistutilsCxxPatch`?

Reply:
@knedlsepp The meaning depends on the context. The point to have this variable in the first place is to query ```python.hasDistutilsCxxPatch``` from numpy so that its numpy counterpart can be engaged (it won't work otherwise). But PR's are welcome if you still want to change it.